### DISCUSSION TITLE
Add support for translations

### DISCUSSION THREAD

**Discussion by Nielsvanpach** [spatie/holidays#131] [Ideas]
We're open to accept a PR on this!

**Comment by herpaderpaldent** [comment]
Hey @Nielsvanpach 

i have created a small proof of concept with translation support: https://github.com/spatie/holidays/pull/173.

While creating it, i have asked myself how much can be shared across languages. For German speaking countries i can imagine most of the holidays name are shared. Although some would differ on a regional level but those could be hard coded. 

Here is a small list of countries with the need of translations that comes into my mind:
- Italy 
- Switzerland
- Canada

What would your prefered way solution look like. Right now my solution is on a national level and relativeley maintainable. Would you like to see some laravel-like `trans()` implementation?

**Reply by Nielsvanpach** [reply]
Hi @herpaderpaldent, thanks for your work on this! I've just added an implementation for multi language countries: https://github.com/spatie/holidays/pull/177
